src/components/generic/Button.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components';
+import { ThemeProvider } from 'styled-components';
+
+// Palette generated @ https://coolors.co/
+export const theme = {
+  start: {
+    bgColor: "#44AF69",
+    color: "#ffffff"
+  },
+  pause: {
+    bgColor: "#FCAB10",
+    color: "#000000"
+  },
+  resume: {
+    bgColor: "#2B9EB3",
+    color: "#ffffff"
+  },
+  fastForward: {
+    bgColor: "#DBD5B5",
+    color: "#000000"
+  },
+  reset: {
+    bgColor: "#F8333C",
+    color: "#ffffff"
+  }
+};
+
+const StyledButton = styled.button`
+  font-size: 17px;
+  text-transform: uppercase;
+  margin: 5px;
+  width: 90%;
+  height: 65px;
+  border-radius: 5px;
+  font-weight: 700;
+  border: 0;
+  letter-spacing: 0.5px;
+  cursor: ${props => {if (!props.disabled) return 'pointer';}};
+  opacity: ${props => {if (props.disabled) return '0.4';}};
+  background-color: ${props => {
+    if (props.type === 'Start') return props.theme.start.bgColor;
```

Comment:
wouldn't use strings here - type shouldn't be the label passed to the button. imagine if you wanted to add a different language.